README.md

```diff
@@ -264,19 +284,24 @@ $ LD_LIBRARY_PATH=/usr/lib/flutter_engine/profile/ ./flutter-client ./sample/bui
 $ LD_LIBRARY_PATH=/usr/lib/flutter_engine/release/ ./flutter-client ./sample/build/linux/x64/release/bundle
 ```
 
-#### Run with DRM backend
+#### Run with DRM backend or EGLStream backend
 
 You need to switch from GUI which is running X11 or Wayland to the Character User Interface (CUI). In addition, `FLUTTER_DRM_DEVICE` must be set properly. The default value is `/dev/dri/card0`.
 
 ```Shell
 $ Ctrl + Alt + F3 # Switching to CUI
+
+# e.g. Run with DRM backend
 $ sudo FLUTTER_DRM_DEVICE="/dev/dri/card1" ./flutter-drm-backend ./sample/build/linux/x64/release/bundle
+
+# e.g. Run with EGLStream backend
+$ sudo FLUTTER_DRM_DEVICE="/dev/dri/card1" ./flutter-eglstream-backend ./sample/build/linux/x64/release/bundle
```

Comment:
It is useless because it is almost duplicate with the above. Please change the program name for general purposes and make it common.

Reply:
I fixed it.